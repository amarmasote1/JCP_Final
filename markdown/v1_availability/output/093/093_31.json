{
    "warnings": null,
    "message": "Success",
    "nodeType": null,
    "body": {
        "dirty": false,
        "inventoryTimestamp": 1583130287000,
        "shipSafetyFactorSource": 0,
        "bopusSafetyFactor": 2,
        "shipSafetyFactor": 1,
        "onhandSupply": 2,
        "reservedDemand": 0,
        "pickSafetyFactorSource": 0,
        "unconfirmedDemand": 1
    },
    "key": "JCP|1037|58078781036|EACH",
    "status": 200
}