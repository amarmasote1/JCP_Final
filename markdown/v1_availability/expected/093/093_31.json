{
	"status": 200,
	"message": "Success",
	"key": "JCP|1037|58078781036|EACH",
	"body": {
		"onhandSupply": 1,
		
		"shipSafetyFactor": 1,
		"bopusSafetyFactor": 2,
		"shipSafetyFactorSource": 0,
		"pickSafetyFactorSource": 0,

		
		
		
		"reservedDemand": 0,
		"unconfirmedDemand": 1,
		
		
		"dirty": false
	},
	"warnings": null
}